lib/Controller/Stats.php

```diff
@@ -346,6 +356,7 @@ public function grid(Request $request, Response $response)
         $type = strtolower($sanitizedQueryParams->getString('type'));
 
         $displayId = $sanitizedQueryParams->getInt('displayId');
+        $displays = $sanitizedQueryParams->getIntArray('displayIds');
```

Comment:
I think we need to provide default value here - an empty array.
That's most likely the reason why the tests failed.